content/tracing/_index.md

```diff
@@ -2,75 +2,201 @@
 title: APM (Tracing)
 kind: Documentation
 description: Instrument your code to improve performance
+further_reading:
+- link: "/tracing/setup"
+  tag: "Documentation"
+  text: Instrument your code
+- link: "/tracing/services"
+  tag: "Documentation"
+  text: Analyse your services
+- link: "/tracing/traces"
+  tag: "Documentation"
+  text: Analyse your traces
 ---
 
 ## Overview
 
 Datadog's integrated APM tool eliminates the traditional separation between infrastructure and application performance monitoring. This not only provides greater visibility, but also allows you to see the relationship between application code and the underlying infrastructure.
 
+Datadog APM is offered as an upgrade to our Pro and Enterprise plans. A free 14-day trial is available.
+Registered users can visit the [APM page of the Datadog application](https://app.datadoghq.com/apm/home) to get started.
+
 ## Getting started
 
-Datadog APM is offered as an upgrade to our Pro and Enterprise plans. A free 14-day trial is available.  Registered users can visit the [APM page of the Datadog application](https://app.datadoghq.com/apm/home) to get started.
+To understand how tracing work, let's take the following example that represent a simple API **thinker-api** and a micro-service behind it **thinker-microservice**, when the api receive a request with the correct *subject* parameter, it answer with a thought, otherwise it answer an error:
```

Comment:
I would rephrase this as: 

```
To understand how tracing work, let's take the following example which represents a simple API **thinker-api** and a micro-service behind it **thinker-microservice**. When the API receives a request with the correct *subject* parameter, it answers with a *thought*, otherwise, it answers an error:
```